reference/enterprise-api-ref/federated-reporting-api.markdown

```diff
@@ -0,0 +1,233 @@
+---
+layout: default
+title: Federated reporting configuration API
+published: true
+tags: [reference, enterprise, API, reporting]
+---
+
+Federated reporting API allows to configure federated reporting process.
```

Comment:
This API is used for configuring hubs so that a single hub can be used to report on any host connected to participating feeder hubs.